SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -31,36 +34,36 @@ void FSpatialTestSettings::Override(const FString& MapName)
 	Duplicate(OriginalSpatialGDKEditorSettings);
 	Duplicate(OriginalGeneralProjectSettings);
 	Duplicate(OriginalEditorPerformanceSettings);
-	// First override the settings from the base config file, if it exists
+
+	// Base config, applies to all maps if present
 	if (FPaths::FileExists(BaseOverridesFilename))
 	{
+		// Override the settings from the base config file
 		Load(BaseOverridesFilename);
 	}
-	else
-	{
-		UE_LOG(LogSpatialTestSettings, Log, TEXT("Base config file not found %s."), *BaseOverridesFilename);
```

Comment:
why remove this?

Reply:
NWX reported log spam from these overrides and as this is base config is optional it will log for every map, and in NWX case every test as they currently have a hook to run the overrides per test as some tests do not have a map.